static/js/collapsible-filters/interactions.js

```diff
@@ -240,4 +241,18 @@ export default class Interactions {
       yxtFooter.classList.add(this.templateName);
     }
   }
+
+  isCollapsibleFiltersView() {
+    const mediaQuery = window.matchMedia(`(min-width: ${variables.cssBreakpointExpandedFilters})`);
+    return !mediaQuery.matches;
+  }
+
+  setListenerOnExpandedFiltersBreakpoint(handleChange) {
```

Comment:
What do you think of `addExpandedFiltersListener`? Also, can we abstract away the event so that the code which calls this function reads like this:
```
collapsibleFiltersInteractions.addExpandedFiltersListener(isExpanded => console.log(isExpanded));
```

Reply:
sure thing